sdk/src/main/java/com/hedera/hashgraph/sdk/PrivateKey.java

```diff
@@ -133,6 +133,16 @@ public static PrivateKey fromBytesDER(byte[] privateKey) {
         return PrivateKey.fromPrivateKeyInfo(PrivateKeyInfo.getInstance(privateKey));
     }
 
+    public static Boolean isED25519(byte[] privateKey) {
```

Comment:
Hmm, I think it might be better to make these methods abstract and by default return `false` and perhaps they can be updated in `PrivateKeyEd25519` and PrivateKeyECDSA` to return `true` for their variant.

cc @Sean-Tedrow-LB